firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -2074,16 +1880,7 @@ class BrowserViewController: UIViewController,
     // Update border to hide when microsurvey is shown so that
     // it appears to belong the app and harder to spoof
     private func updateBarBordersForMicrosurvey() {
```

Comment:
This function is just a guard now, I think we can remove it?

Reply:
Should be good to remove since we update the borders here for the new toolbar redesign:

<img width="982" height="785" alt="image" src="https://github.com/user-attachments/assets/9273d17c-4fc5-4cc8-8922-18a274ffd415" />

Reply:
@cyndichin I'll let you take care of it then. Thank you 🙏